config.yaml

```diff
@@ -3,8 +3,8 @@ dynamo-endpoint: dynamodb.us-east-1.amazonaws.com
 
 # Approved Application Authentication Credentials
 approved-keys:
-  - application:
-    secret:
+  - application: lightning
+    secret: secret
```

Comment:
Can you remove these so that we keep the `config.yaml` that's in the repo as a template?